frontend/src/scenes/max/maxGlobalLogic.ts

```diff
@@ -27,7 +31,7 @@ export interface ToolDefinition {
     /** Optional: When in context, the tool can add items to the pool of Max's suggested questions */
     suggestions?: string[] // TODO: Suggestions aren't used yet, pending a refactor of maxLogic's allSuggestions
     /** The callback function that will be executed with the LLM's tool call output */
-    callback: (toolOutput: any) => void
+    callback: (toolOutput: any, continueGeneration: () => void) => void | Promise<void>
```

Comment:
shouldn't this be an optional argument, that we would only call for the `navigate` tool? Other tools won't have interrupt nodes in the backend by default.